app/models/apple/config.rb

```diff
@@ -6,32 +6,23 @@ class Config < ApplicationRecord
     DEFAULT_TITLE = "Apple Delegated Delivery Subscriptions"
     DEFAULT_AUDIO_FORMAT = {"f" => "flac", "b" => 16, "c" => 2, "s" => 44100}.freeze
 
-    belongs_to :podcast, class_name: "Podcast"
-    belongs_to :public_feed, class_name: "Feed"
-    belongs_to :private_feed, class_name: "Feed"
-    belongs_to :key, class_name: "Apple::Key", optional: true
+    belongs_to :feed
```

Comment:
So the only feed associated is the private apple feed, and I assume when I get to the migrations you are renaming that as `feed_id`